code/src/main/kotlin/com/expediagroup/sdk/lodgingconnectivity/graphql/model/exception/NoDataException.kt

```diff
@@ -0,0 +1,9 @@
+package com.expediagroup.sdk.lodgingconnectivity.graphql.model.exception
+
+import com.expediagroup.sdk.lodgingconnectivity.graphql.model.response.RawResponseError
+
+class NoDataException(
```

Comment:
Let's make this extend our EG Service Exception.

Reply:
Done